### PR TITLE
Feature/plib 2509  html example list

### PR DESCRIPTION
**Pull request by kevinmpowell**
`npm run dev` and view `/proving-grounds/html-example-list`

### CODE REVIEW

File: docs/.vuepress/components/cdr-doc-html-example-list.vue [hunk 1]
```diff
@@ -0,0 +1,242 @@
+<template>
+  <div class="cdr-doc-html-example-list" :class="{'cdr-doc-html-example-list--not-interactive': !interactive }">
```
Comment (mhewson) [inline docs/.vuepress/components/cdr-doc-html-example-list.vue]: can you give me some background on why this is described as a list? it does not seem to be one in anyway that I can understand

File: docs/proving-grounds/html-example-list/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+```html
+  <cdr-button size="large">Default</cdr-button>
+```
+  
+  </template>
+  <template slot="Hover">
+
+```html
+  <cdr-button size="small" class="cdr-doc-button-demo--hover">Hover</cdr-button>
```
Comment (mhewson) [inline docs/proving-grounds/html-example-list/README.md]: I'm not a fan of maintaining doc classes that set false states to component interactions, at best they provide duplicative experience to the actual component, at worst we have something that now has to be versioned or at least maintained prior to an update that has not proven to be helpful to the consumer

File: docs/.vuepress/components/cdr-doc-html-example-list.vue (hunk 1 above)
Comment (kevinmpowell) [reply docs/.vuepress/components/cdr-doc-html-example-list.vue]: @mhewson did you still want to change the name to something else after our conversation yesterday? If so, any suggestions?

File: docs/proving-grounds/html-example-list/README.md (hunk 2 above)
Comment (kevinmpowell) [reply docs/proving-grounds/html-example-list/README.md]: This is an example of how the component could be used to show states of various components. This component handles the display of multiple examples, and the proving ground is not a prescription for how or even if it should be used this way.

File: docs/.vuepress/components/cdr-doc-html-example-list.vue (hunk 1 above)
Comment (mhewson) [reply docs/.vuepress/components/cdr-doc-html-example-list.vue]: It's not my fav but really not a big priority

**Pull request merged**